### PR TITLE
Changelog from 4.7.0

### PR DESCRIPTION
**Pull request by pthoang**
## Description
A branch to keep tabs of the changes merged into develop

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -58,3 +58,38 @@
   - Fixed issues with Sheet creating duplicate objects of same instance
 - Topbar
   - Fixed issues with Topbar producing unnecessary EventListeners
+
+### Next release
+- Sidebar
+  - Fixed sidebar positioning making some of the content of sidebar not visible in some cases
+- Steps
+  - Fixed vertical clickable steps misplacement of visual elements
+- Design guide
```
Comment (payex-dev) [inline CHANGELOG.md:67]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310&bid=19786438)

File: CHANGELOG.md [hunk 2]
```diff
@@ -58,3 +58,38 @@
...
+- Datepicker
+  - Fixed datepicker with time not showing time component immediately when default value is not provided 
+- Forms
+  - Radio buttons
+    - Added visual evident hover and focus styling
+    - Made using radio buttons without label content possible
+  - Checkboxes
+    - Added visual evident hover and focus styling
+    - Made using radio buttons without label content possible
+- Tabs
```
Comment (payex-dev) [inline CHANGELOG.md:92]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

File: CHANGELOG.md [hunk 3]
```diff
@@ -58,3 +58,38 @@
...
+  - Fixed sidebar positioning making some of the content of sidebar not visible in some cases
+- Steps
+  - Fixed vertical clickable steps misplacement of visual elements
+- Design guide
+  - Added better alt text to logos in Design Guide
+  - Fixed margin right of documentation content container
+  - Changed styling of the sidebar, moved the contact part to the bottom
+  - Added aria-hidden to all icons
+  - Fixed typos
+  - Grid
```
Comment (payex-dev) [inline CHANGELOG.md:73]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

File: CHANGELOG.md [hunk 4]
```diff
@@ -58,3 +58,38 @@
...
+    - Fixed grid gutter not being updated to 32px in documentation
+  - Status
+    - Added example using status in Status documentation
+- Status
```
Comment (payex-dev) [inline CHANGELOG.md:77]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310&bid=19786438)

File: CHANGELOG.md [hunk 5]
```diff
@@ -58,3 +58,57 @@
...
+  - Checkboxes
+    - Added visual evident hover and focus styling
+    - Made using radio buttons without label content possible
+  - Input/Text fields
+    - Fixed issue with error icon not meshing well with the text underneath
+- Tabs
+  - Changed styling of tabs
+- Slab
+  - Added slab with top border
+- Accordion
```
Comment (payex-dev) [inline CHANGELOG.md:103]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

File: CHANGELOG.md [hunk 6]
```diff
@@ -58,3 +58,57 @@
   - Fixed issues with Sheet creating duplicate objects of same instance
 - Topbar
   - Fixed issues with Topbar producing unnecessary EventListeners
+
+### Next release
+- Sidebar
+  - Fixed sidebar positioning making some of the content of sidebar not visible in some cases
+- Steps
+  - Fixed vertical clickable steps misplacement of visual elements
+- Design guide
```
Comment (payex-dev) [inline CHANGELOG.md:67]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

File: CHANGELOG.md [hunk 7]
```diff
@@ -58,3 +58,57 @@
...
+  - Added new component to present DG components
+  - Grid
+    - Fixed grid gutter not being updated to 32px in documentation
+  - Status
+    - Added example using status in Status documentation
+  - Buttons
+    - Changed guidelines content and presentation
+  - Forms
+    - Updated documentation, moved dropdown menu to Forms
+- Status
```
Comment (payex-dev) [inline CHANGELOG.md:82]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

File: CHANGELOG.md [hunk 8]
```diff
@@ -58,3 +58,57 @@
...
+- Forms
+  - Radio buttons
+    - Added visual evident hover and focus styling
+    - Made using radio buttons without label content possible
+  - Checkboxes
+    - Added visual evident hover and focus styling
+    - Made using radio buttons without label content possible
+  - Input/Text fields
+    - Fixed issue with error icon not meshing well with the text underneath
+- Tabs
```
Comment (payex-dev) [inline CHANGELOG.md:99]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

File: CHANGELOG.md [hunk 9]
```diff
@@ -58,3 +58,57 @@
...
+- Steps
+  - Fixed vertical clickable steps misplacement of visual elements
+- Design guide
+  - Added better alt text to logos in Design Guide
+  - Fixed margin right of documentation content container
+  - Changed styling of the sidebar, moved the contact part to the bottom
+  - Added aria-hidden to all icons
+  - Fixed typos
+  - Added new component to present DG components
+  - Grid
```
Comment (payex-dev) [inline CHANGELOG.md:74]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310&bid=19786438)

File: CHANGELOG.md [hunk 10]
```diff
@@ -58,3 +58,61 @@
...
+  - Checkboxes
+    - Added visual evident hover and focus styling
+    - Made using radio buttons without label content possible
+  - Input/Text fields
+    - Fixed issue with error icon not meshing well with the text underneath
+- Tabs
+  - Changed styling of tabs
+- Slab
+  - Added slab with top border
+- Accordion
```
Comment (payex-dev) [inline CHANGELOG.md:105]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

File: CHANGELOG.md [hunk 11]
```diff
@@ -58,3 +58,61 @@
...
+- Steps
+  - Fixed vertical clickable steps misplacement of visual elements
+- Design guide
+  - Added better alt text to logos in Design Guide
+  - Fixed margin right of documentation content container
+  - Changed styling of the sidebar, moved the contact part to the bottom
+  - Added aria-hidden to all icons
+  - Fixed typos
+  - Added new component to present DG components
+  - Grid
```
Comment (payex-dev) [inline CHANGELOG.md:74]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

File: CHANGELOG.md [hunk 12]
```diff
@@ -58,3 +58,61 @@
...
+    - Fixed grid gutter not being updated to 32px in documentation
+  - Status
+    - Added example using status in Status documentation
+  - Buttons
+    - Changed guidelines content and presentation
+  - Forms
+    - Updated documentation, moved dropdown menu to Forms
+  - Status Badges
+    - Added status badges that give information on component status
+- Status
```
Comment (payex-dev) [inline CHANGELOG.md:84]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

File: CHANGELOG.md [hunk 13]
```diff
@@ -58,3 +58,61 @@
   - Fixed issues with Sheet creating duplicate objects of same instance
 - Topbar
   - Fixed issues with Topbar producing unnecessary EventListeners
+
+### Next release
+- Sidebar
+  - Fixed sidebar positioning making some of the content of sidebar not visible in some cases
+- Steps
+  - Fixed vertical clickable steps misplacement of visual elements
+- Design guide
```
Comment (payex-dev) [inline CHANGELOG.md:67]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

File: CHANGELOG.md [hunk 14]
```diff
@@ -58,3 +58,61 @@
...
+- Forms
+  - Radio buttons
+    - Added visual evident hover and focus styling
+    - Made using radio buttons without label content possible
+  - Checkboxes
+    - Added visual evident hover and focus styling
+    - Made using radio buttons without label content possible
+  - Input/Text fields
+    - Fixed issue with error icon not meshing well with the text underneath
+- Tabs
```
Comment (payex-dev) [inline CHANGELOG.md:101]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 23
- Added 23
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310&bid=19786438)

File: CHANGELOG.md [hunk 15]
```diff
@@ -3,58 +3,69 @@
...
-- Sidebar
-  - Changed leaf styling when used on same level as subgroups
-- Steps
-  - Renamed steps in documentation
-  - Updated styling
-- Topbar
-  - Reversed icon of topbar-link-right
+  - Radio buttons
+    - Added visual evident hover and focus styling
+    - Made using radio buttons without label content possible
```
Comment (payex-dev) [inline CHANGELOG.md:34]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

File: CHANGELOG.md [hunk 16]
```diff
@@ -3,58 +3,69 @@
...
+  - Added slab with top border
+- Status
+  - Added a second warning status
+- Toolips
+  - Added extended tooltips that adjust when falling out of the viewport
 - Design Guide
-  - Button
-    - Added .btn-link recommendation
+  - Added better alt text to logos in Design Guide
+  - Added aria-hidden to all icons
```
Comment (payex-dev) [inline CHANGELOG.md:18]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

File: CHANGELOG.md [hunk 17]
```diff
@@ -3,58 +3,69 @@
...
+  - Added new component to present DG components
+  - Status
+    - Added example using status in Status documentation
+  - Status Badges
+    - Added status badges that give information on component status
```
Comment (payex-dev) [inline CHANGELOG.md:23]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

File: CHANGELOG.md [hunk 18]
```diff
@@ -3,58 +3,69 @@
...
-  - Made icons top-align instead of center align verically
-- Sidebar
-  - Changed leaf styling when used on same level as subgroups
-- Steps
-  - Renamed steps in documentation
-  - Updated styling
-- Topbar
-  - Reversed icon of topbar-link-right
+  - Radio buttons
+    - Added visual evident hover and focus styling
```
Comment (payex-dev) [inline CHANGELOG.md:33]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

File: CHANGELOG.md (hunk 16 above)
Comment (payex-dev) [inline CHANGELOG.md:18]: Codacy found an issue: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

File: CHANGELOG.md [hunk 19]
```diff
@@ -3,58 +3,69 @@
...
+    - Made using radio buttons without label content possible
+  - Checkboxes
+    - Added visual evident hover and focus styling
+    - Made using radio buttons without label content possible
```
Comment (payex-dev) [inline CHANGELOG.md:37]: Codacy found an issue: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

File: CHANGELOG.md [hunk 20]
```diff
@@ -3,58 +3,69 @@
...
+- Logotype
+  - Merged Logos and Logotype into Logotype under Core
+  - Updated preview of Logotype examples and updated documentation
+- Status
+  - Removed default status
```
Comment (payex-dev) [inline CHANGELOG.md:42]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

File: CHANGELOG.md [hunk 21]
```diff
@@ -3,58 +3,69 @@
...
+- Toolips
+  - Added extended tooltips that adjust when falling out of the viewport
 - Design Guide
-  - Button
-    - Added .btn-link recommendation
+  - Added better alt text to logos in Design Guide
+  - Added aria-hidden to all icons
+  - Added new component to present DG components
+  - Status
+    - Added example using status in Status documentation
```
Comment (payex-dev) [inline CHANGELOG.md:21]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

File: CHANGELOG.md [hunk 22]
```diff
@@ -3,58 +3,69 @@
...
+- Buttons
+  - Added destructive button
+- Cards
+  - Added new design of cards
+- Slab
+  - Added slab with top border
+- Status
+  - Added a second warning status
+- Toolips
+  - Added extended tooltips that adjust when falling out of the viewport
```
Comment (payex-dev) [inline CHANGELOG.md:15]: Codacy found an issue: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

File: CHANGELOG.md [hunk 23]
```diff
@@ -3,58 +3,69 @@
...
-  - Reversed icon of topbar-link-right
+  - Radio buttons
+    - Added visual evident hover and focus styling
+    - Made using radio buttons without label content possible
+  - Checkboxes
+    - Added visual evident hover and focus styling
+    - Made using radio buttons without label content possible
+- Logotype
+  - Merged Logos and Logotype into Logotype under Core
+  - Updated preview of Logotype examples and updated documentation
```
Comment (payex-dev) [inline CHANGELOG.md:40]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

File: CHANGELOG.md [hunk 24]
```diff
@@ -3,58 +3,69 @@
...
+  - Input/Text fields
+    - Fixed issue with error icon not meshing well with the text underneath
+- Sidebar
+  - Fixed sidebar positioning making some of the content of sidebar not visible in some cases
+- Steps
+  - Fixed vertical clickable steps misplacement of visual elements
+- Design Guide
+  - Fixed margin right of documentation content container
+  - Fixed typos
+  - Grid
```
Comment (payex-dev) [inline CHANGELOG.md:70]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

File: CHANGELOG.md [hunk 25]
```diff
@@ -3,58 +3,69 @@
...
+  - Status
+    - Added example using status in Status documentation
+  - Status Badges
+    - Added status badges that give information on component status
 
 ### Changed
-- Breakpoints
-  - Moved to Utilities
+- Accordion
+  - Changed padding
```
Comment (payex-dev) [inline CHANGELOG.md:27]: Codacy found an issue: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

File: CHANGELOG.md [hunk 26]
```diff
@@ -3,58 +3,69 @@
...
-- Topbar
-  - Reversed icon of topbar-link-right
+  - Radio buttons
+    - Added visual evident hover and focus styling
+    - Made using radio buttons without label content possible
+  - Checkboxes
+    - Added visual evident hover and focus styling
+    - Made using radio buttons without label content possible
+- Logotype
+  - Merged Logos and Logotype into Logotype under Core
```
Comment (payex-dev) [inline CHANGELOG.md:39]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

File: CHANGELOG.md [hunk 27]
```diff
@@ -3,58 +3,69 @@
...
+- Slab
+  - Added slab with top border
+- Status
+  - Added a second warning status
+- Toolips
+  - Added extended tooltips that adjust when falling out of the viewport
 - Design Guide
-  - Button
-    - Added .btn-link recommendation
+  - Added better alt text to logos in Design Guide
```
Comment (payex-dev) [inline CHANGELOG.md:17]: Codacy found an issue: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

Comment (payex-dev) [inline CHANGELOG.md:17]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

File: CHANGELOG.md [hunk 28]
```diff
@@ -3,58 +3,69 @@
...
-  - Made Topbar menu able to have sidebar content, tweaked Sidebar styling when inside of Topbar
+- Buttons
+  - Added destructive button
+- Cards
+  - Added new design of cards
+- Slab
+  - Added slab with top border
+- Status
+  - Added a second warning status
+- Toolips
```
Comment (payex-dev) [inline CHANGELOG.md:14]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

File: CHANGELOG.md (hunk 23 above)
Comment (payex-dev) [inline CHANGELOG.md:40]: Codacy found an issue: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

File: CHANGELOG.md (hunk 25 above)
Comment (payex-dev) [inline CHANGELOG.md:27]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

File: CHANGELOG.md [hunk 29]
```diff
@@ -3,58 +3,69 @@
...
 
 ### Changed
-- Breakpoints
-  - Moved to Utilities
+- Accordion
+  - Changed padding
 - Buttons
-  - Updated button sizes
+  - Changed color of buttons
+  - Renamed buttons
```
Comment (payex-dev) [inline CHANGELOG.md:30]: Codacy found an issue: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

File: CHANGELOG.md [hunk 30]
```diff
@@ -3,58 +3,69 @@
...
+    - Made using radio buttons without label content possible
+  - Checkboxes
+    - Added visual evident hover and focus styling
+    - Made using radio buttons without label content possible
+- Logotype
+  - Merged Logos and Logotype into Logotype under Core
+  - Updated preview of Logotype examples and updated documentation
+- Status
+  - Removed default status
+  - Changed color of success status
```
Comment (payex-dev) [inline CHANGELOG.md:43]: Codacy found an issue: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

File: CHANGELOG.md [hunk 31]
```diff
@@ -3,58 +3,69 @@
...
-  - Updated Action Link for new tab color
-  - Made icons top-align instead of center align verically
-- Sidebar
-  - Changed leaf styling when used on same level as subgroups
-- Steps
-  - Renamed steps in documentation
-  - Updated styling
-- Topbar
-  - Reversed icon of topbar-link-right
+  - Radio buttons
```
Comment (payex-dev) [inline CHANGELOG.md:32]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

File: CHANGELOG.md (hunk 21 above)
Comment (payex-dev) [inline CHANGELOG.md:21]: Codacy found an issue: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

File: CHANGELOG.md [hunk 32]
```diff
@@ -3,58 +3,69 @@
...
+- Status
+  - Added a second warning status
+- Toolips
+  - Added extended tooltips that adjust when falling out of the viewport
 - Design Guide
-  - Button
-    - Added .btn-link recommendation
+  - Added better alt text to logos in Design Guide
+  - Added aria-hidden to all icons
+  - Added new component to present DG components
```
Comment (payex-dev) [inline CHANGELOG.md:19]: Codacy found an issue: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=6038310)

**Review by VikingTristan — APPROVED** [review]
Ignoring codacy markdown rules for this round.

**Pull request merged**